multipart-form/templates/page.mustache

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>HTMLForm</title>
+    <meta charset="UTF-8">
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body class="bg-gray-100">
+    <div class="text-center">
+        <h2 class="text-3xl p-3">Multipart Form</h2>
+    </div>
+    <div class="p-6 max-w-md mx-auto bg-white rounded-xl items-centered space-x-4 shadow-lg">
+        <div>
+            <img src="images/hummingbird.png" class="w-64 mx-auto"/>
+        </div>
+        {{$body}}
+        {{/body}}
```

Comment:
Is this right? I don't know enough about mustache

Reply:
Yes. body comes from the page that referenced this file. See the section on blocks in https://mustache.github.io/mustache.5.html